.circleci/circle_vm_setup.sh

```diff
@@ -34,9 +32,7 @@ echo "export PATH=/home/linuxbrew/.linuxbrew/bin:$PATH" >>~/.bashrc
 . ~/.bashrc
 
 brew update && brew tap drud/ddev
-for item in mkcert ddev composer php docker-compose; do
-    brew install $item || /home/linuxbrew/.linuxbrew/bin/brew upgrade $item
-done
+brew install mkcert ddev composer php docker-compose
```

Comment:
Looks like it's failing because it's trying to compile php 7.4 as the default. I guess we should bump from 7.2 to 7.3, but probably not 7.4 at this point?